CHANGELOG.md

```diff
@@ -4,6 +4,37 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v6.0.0](https://github.com/voxpupuli/puppet-redis/tree/v6.0.0) (2020-05-11)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-redis/compare/v5.0.0...v6.0.0)
+
+**Breaking changes:**
+
+- Make apt and epel soft dependencies [\#358](https://github.com/voxpupuli/puppet-redis/pull/358) ([ekohl](https://github.com/ekohl))
+- Require puppet-epel over stahnma-epel [\#352](https://github.com/voxpupuli/puppet-redis/pull/352) ([dhoppe](https://github.com/dhoppe))
```

Comment:
Does it make sense to keep this in now that the change above it was merged?

Reply:
No